src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -1470,7 +1470,10 @@ If such a class is used as a base class and if the deriving class defines a dest
     <value>Predefined type '{0}' is not defined or imported</value>
   </data>
   <data name="ERR_PredefinedValueTupleTypeNotFound" xml:space="preserve">
-    <value>Predefined type '{0}' is not defined or imported, or is declared in multiple referenced assemblies</value>
+    <value>Predefined type '{0}' is not defined or imported</value>
```

Comment:
><value>Predefined type '{0}' is not defined or imported</value> [](start = 4, length = 63)

It looks like the error message matches the one above. Do we need two distinct error numbers for them?

Reply:
We do. The ValueTuple error triggers a codefix in the IDE (AddNuget).

Reply:
I can add a comment